src/app/components/blog/services/form-fields.service.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { TestBed } from '@angular/core/testing';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
```

Comment:
I love that you have tests! Can we name this file something that has the word Test in it though so that it is clear? Is that normal practice for Angular?